lib/jsonapi_compliable/resource.rb

```diff
@@ -24,6 +24,35 @@ def context_namespace
       JsonapiCompliable.context[:namespace]
     end
 
+    # return single/raise if not found
+    # EmployeeResource.find(params)
+    #
+    # return single/DONT raise if not found
+    #
+    # return multiple
+    # EmployeeResource.all(params) # base_scope if needed
+    #
+    # posts = EmployeeResource.query(params)
+    # render jsonapi: posts
+    #
+    # maybe
+    # EmployeeResource.create(params)
+    # EmployeeResource.update(params)
+    # EmployeeResource.destroy(params)
+    def self.all(params, base_scope = nil)
+      runner = Runner.new(self, params)
+      # todo resource base scope
+      runner.resolve(base_scope || model.all)
+    end
+
+    def self.find(params, base_scope = nil)
+      params[:filter] ||= {}
+      params[:filter].merge!(id: params[:id])
```

Comment:
Assuming that at least the AR adapter has id as filterable by default?

Reply:
everyone gets `attribute :id, :string` by default (you can do it explicitly to override)